nomad-connect-acls/05-nomad_config.md

```diff
@@ -2,7 +2,7 @@
 .lang-screenshot { -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none; }
 </style>
 
-Open the `nomad_config.hcl`{{open}} file. Add a consul stanza with your token.
+Open the `nomad_config.hcl`{{open}} file. Add a Consul stanza with your token.
```

Comment:
I had it lowercased here because it is the proper name of the stanza.  Perhaps we can go back to downcase and put it in backticks?
```suggestion
Open the `nomad_config.hcl`{{open}} file. Add a `consul` stanza with your token.
```